checkov/kubernetes/runner.py

```diff
@@ -4,49 +4,58 @@
 from functools import reduce
 
 from checkov.common.bridgecrew.platform_integration import bc_integration
-from checkov.common.parallelizer.parallel_runner import parallel_runner
+from checkov.common.checks_infra.registry import get_graph_checks_registry
+from checkov.common.graph.db_connectors.networkx.networkx_db_connector import NetworkxConnector
+from checkov.common.graph.graph_builder import CustomAttributes
 from checkov.common.util.data_structures_utils import search_deep_keys
 from checkov.common.util.type_forcers import force_list
 from checkov.common.output.record import Record
 from checkov.common.output.report import Report
-from checkov.common.runners.base_runner import BaseRunner, filter_ignored_paths
-from checkov.kubernetes.parser.parser import parse
+from checkov.common.runners.base_runner import BaseRunner
+from checkov.kubernetes.graph_builder.local_graph import KubernetesLocalGraph
+from checkov.kubernetes.graph_manager import KubernetesGraphManager
+from checkov.kubernetes.kebernetes_utils import get_files_definitions, get_folder_definitions
 from checkov.kubernetes.checks.resource.registry import registry
 from checkov.runner_filter import RunnerFilter
 
-K8_POSSIBLE_ENDINGS = [".yaml", ".yml", ".json"]
-
 
 class Runner(BaseRunner):
-    check_type = "kubernetes"
+    def __init__(
+        self,
+        graph_class=KubernetesLocalGraph,
+        db_connector=NetworkxConnector(),
+        source="Kubernetes",
+        graph_manager=None,
+    ):
+        self.check_type = "kubernetes"
+        self.graph_class = graph_class
+        # TODO: uncomment it in order to create the graph
```

Comment:
Why is this commented out?

Reply:
I'll do it in the next task, and already implemented some staff so I don't want to delete